README.md

```diff
@@ -1,2 +1,48 @@
-# video-store-cli
-A Command Line Interface for Retro Video Store API
+# Video Store CLI
+
+Once upon a time in ye olden days to watch a movie a person had to head down to their local video store and rent a video.  We are going to step into the shoes of this old timey retro video store owner and build for ourselves all of the tools that we need in order to run our successful corner video store. 
+
+You have already built a [Video Store API](https://github.com/AdaGold/retro-video-store) for the backend in Part 1 of the project.
+
+In keeping with our retro vibe, for this part of the project we will build a Command Line Interface (CLI) for the front end.
+
+## Manual Tests
+
+We will be testing your CLI manually to see how it performs all of the required behaviors.  We recommend that you find a partner to "play-test" your CLI and put it through it's paces. 
+
+## Timeline
+
+This project's submission deadline is Friday May 28th, 9:00PM.
```

Comment:
As I mentioned in the other review, I'm in favor of taking this out until/unless we agree collectively that we want to start having deadlines in READMEs again.

Reply:
I did not realize we didn't usually including dates in READMEs, but now that you mention it that makes sense, I think taking it out is a good plan :-).

Reply:
To be fair it's a new thing this cohort.

Reply:
```suggestion
```